cmd/argo/commands/get.go

```diff
@@ -61,13 +63,25 @@ func NewGetCommand() *cobra.Command {
 		Use:   "get WORKFLOW...",
 		Short: "display details about a workflow",
 		Run: func(cmd *cobra.Command, args []string) {
-			if len(args) == 0 {
+			if (len(args) == 0 && !getArgs.latest) || (len(args) > 0 && getArgs.latest) {
 				cmd.HelpFunc()(cmd, args)
 				os.Exit(1)
 			}
+
 			ctx, apiClient := client.NewAPIClient()
 			serviceClient := apiClient.NewWorkflowServiceClient()
 			namespace := client.Namespace()
+
+			if getArgs.latest {
+				var workflows wfv1.ByCreationTimestamp
+				wfList, err := serviceClient.ListWorkflows(ctx, &workflowpkg.WorkflowListRequest{Namespace: namespace})
+				errors.CheckError(err)
+				workflows = append(workflows, wfList.Items...)
+				sort.Sort(workflows)
```

Comment:
Why do we need to sort here if we're only looking for the min value? We can determine the most recent Workflow in linear time.

Reply:
what if there are no workflow?

Reply:
@simster7 I didn't think of that, good point. I'll change that - should I leave the `ByCreationTimestamp` sorting interface in `workflow_types.go` or remove it entirely?

@alexec Thanks for pointing that out! I will fix that as well

Reply:
I'd remove it.